api/src/main/java/org/openmrs/module/referencemetadata/OrderFrequencyHandler.java

```diff
@@ -0,0 +1,31 @@
+package org.openmrs.module.referencemetadata;
+
+import org.openmrs.OrderFrequency;
+import org.openmrs.annotation.Handler;
+import org.openmrs.api.OrderService;
+import org.openmrs.module.metadatadeploy.handler.AbstractObjectDeployHandler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+@Handler(supports = {OrderFrequency.class})
+public class OrderFrequencyHandler extends AbstractObjectDeployHandler<OrderFrequency> {
```

Comment:
This class already exists in metadatadeploy: https://github.com/openmrs/openmrs-module-metadatadeploy/blob/1.10.0/api-1.10/src/main/java/org/openmrs/module/metadatadeploy/handler/impl/OrderFrequencyDeployHandler.java

You would need to update the version of metadatadeploy, and add a dependency on its api-1.10 in pom.xml